### DISCUSSION TITLE
Make openrct2-cli `--version` flag print the network version

### DISCUSSION THREAD

**Discussion by redthirten** [OpenRCT2/OpenRCT2#16408] [Feature requests] [answered]
The `--version` flag should be more verbose and print the network version of the CLI app, or at least the sub-version (ie. 0.3.5.1 **-9**) and build for easier debugging/identification.

Currently it only prints the main version number and OS.

**Comment by duncanspumpkin** [comment]
You could display all the version numbers, Park Format, Scripting, Network.

**Comment by janisozaur** [comment, marked as answer]
Made a PR: https://github.com/OpenRCT2/OpenRCT2/pull/16413

**Reply by redthirten** [reply]
Thanks! This is perfect!
